packages/server/bin/cli

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env node
+'use strict'
+const path = require('path')
+const yargs = require('yargs')
+require('../bootstrap')
+
+const execution = yargs
```

Comment:
I had an issue where the `knex` CLI couldn't generate new migrations in the proper `module` directory, it just doesn't support choosing a target directory. Additionally I also remember Gergo talking about occasionally creating new scripts for random testing/debugging and needing to build separate debugging launch profiles for each of them.

To solve both issues I added `yargs` which allows for easy/fast CLI building. Currently I've defined 3 commands for it: migrating up, migrating down and adding a new migration into the correct module, but we can now add all kinds of CLI commands using `yargs`